pkg/controllers/managementuser/rbac/prtb_handler.go

```diff
@@ -128,13 +137,63 @@ func (p *prtbLifecycle) syncPRTB(binding *v3.ProjectRoleTemplateBinding) error {
 	return p.reconcileProjectAccessToGlobalResources(binding, roles)
 }
 
+func (p *prtbLifecycle) ensureProjectResourcesRoleBinding(binding *v3.ProjectRoleTemplateBinding) error {
```

Comment:
You are probably going to need similar logic in the CRTB handler, or users who have CRTB access to a cluster may not have the rbac necessary to use this endpoint. Correct me if you disagree.

Reply:
You are right, adding

Reply:
It turns out this wreaks havoc on steve when a user has a combination of CRTBs and PRTBs or CRTBs and regular rolebindings. If a user has a cluster-member CRTB and a project-owner PRTB and tries to list secrets, steve sees the generic CRTB and forwards the request to the global endpoint for secrets, even though the user only has access to secrets in one project. Will have to rethink this, and probably go back to creating custom roles and individual clusterroles and roles for different CRTBs and PRTBs.